shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md

```diff
@@ -7,11 +7,13 @@ Source: [TW](https://archive.org/details/in.ernet.dli.2015.495556)
 
 TODO: परिष्करणीयम्
 
-भूमिका, 
+श्रीः.
+
+भूमिका. 
```

Comment:
Thanks - do a global replacement of . with ।

Reply:
Done.  Please check.